### ISSUE TITLE
Definition list inside of blockquote doesn't work

### ISSUE THREAD

**Issue by justinh-kairos** [wataru-chocola/micromark-extension-definition-list#55]
See [this issue](https://github.com/wataru-chocola/remark-definition-list/issues/39): this extension breaks when a definition list is nested inside of a blockquote element, like so:
```
> ## Title
> 
> Term 1
> : Definition 1
> Term 2
> : Definition 2a
> : Definition 2b
```

**Comment by wataru-chocola** [comment]
@justinh-kairos Thanks for your bug report.

I've just published a new version v1.2.0.
Cloud you try it?

---

By the way, `Term 2` must be followed by a blank line.

```
> Term 1
> : Definition 1
>
> Term 2
> : Definition 2a
> : Definition 2b
```


Spec is here: https://michelf.ca/projects/php-markdown/extra/#def-list

> Terms must be separated from the previous definition by a blank line.


Thanks.

**Comment by justinh-kairos** [comment]
@wataru-chocola Looks like that worked well, thanks for the update!
